examples/FloatShield/FloatShield_OpenLoop/FloatShield_OpenLoop.ino

```diff
@@ -1,19 +1,55 @@
-#include <FloatShield.h>           // Including the FloatShield library
+/*
+  FloatShield Open Loop
 
-void setup() {                     // Setup - runs only once
-    Serial.begin(9600);            // Begin serial comunication
-    FloatShield.begin();           // Initialise FloatShield
-    FloatShield.calibrate();       // Calibrate FloatShield
+  Performs continuous sensor readings with manual fan speed control.
+
+  The example initializes the FloatShield and performs
+  calibration. Then it reads the current position
+  of the potentiometer and sets the fan speed accordingly.
+  The current position of the potentiometer and current
+  position of the ball is periodically (with specified
+  sampling period) sent out through the serial communication,
+  and can be displayed using Serial Plotter tool of Arduino IDE.
+
+  This code is part of the AutomationShield hardware and software
+  ecosystem. Visit http://www.automationshield.com for more
+  details. This code is licensed under a Creative Commons
+  Attribution-NonCommercial 4.0 International License.
+
+  Created by Gergely Takács and Peter Chmurčiak.
+  Last update: 25.06.2019.
+*/
+
+#include <FloatShield.h>              // Including the FloatShield library
+#include <Sampling.h>                 // Include sampling library
+
+unsigned long Ts = 30000;             // Sampling period in microseconds
```

Comment:
I can't really judge just by looking, this does work? So like 30 ms? I'm ok with it, sounds quite reasonable actually.

Reply:
I am writing it like this because function Sampling.period(Ts); takes input in microseconds. Alternative would be for example Ts = 30; Sampling.period(Ts * 1000);

Reply:
I know, I wrote that:) I have no problem with the code itself!

Just wondering about the time 30 ms and how you chose it. And whether faster would not be better for the dynamics.

Reply:
Ah... Well.. To be honest it was just a wild guess based on your approximation https://github.com/gergelytakacs/AutomationShield/pull/165#commitcomment-34066511. So far I did not feel that this sampling period is insufficient. I shall try to change it and see what happens.

Reply:
No worries, just wondering. Engineering is precision guesswork anyways:) It would be possibly interesting to try going lower, e.g. to 10 and see if it improves control later. But your guess seems to be correct so far.